starters/pamelalozano/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>false</threadsafe>
```

Comment:
You can probably make this `<threadsafe>true</threadsafe>`

Reply:
Done.

Reply:
Done.